climateconnect/climateconnect
# Fix: Omit manage members button when creating an organization

## Description
Solves #1009

- Removed manage members button from the edit organization interface.
- When creating an organization, the manage members interface will be opened after they click publish.
- Cancel button text is conditionally changed depending on whether or not the user is coming from the creation stage
- Updated alert message

## Test plan

1. Sign in
2. Go to create organization
3. Go to the detailed organization information portion, enter the necessary fields
4. Press publish
5. Changes should be visible on the manage members page

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
